doajtest/unit/test_background.py

```diff
@@ -0,0 +1,84 @@
+from doajtest.helpers import DoajTestCase
+from doajtest.fixtures import BackgroundFixtureFactory
+
+from portality import models
+from portality.background import BackgroundApi
+
+import time
+
+class TestBackground(DoajTestCase):
+
+    def setUp(self):
+        super(TestBackground, self).setUp()
+
+    def tearDown(self):
+        super(TestBackground, self).tearDown()
+
+    def test_01_with_user_success(self):
+        acc = models.Account()
+        acc.set_id("testuser")
+        acc.save()
+        time.sleep(2)
+
+        task = BackgroundFixtureFactory.get_task(acc.id)
+        BackgroundApi.execute(task)
+        time.sleep(2)
```

Comment:
So this is just to let ES update. Since it's difficult to specify a blocking save here (you would have to change what BackgroundApi.execute does just for the benefit of a faster unit test) it's OK to leave this line as-is. Where we do use `.save()` explicitly in tests we should probably make it `.save(differentiate=True)`.